src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -502,6 +503,15 @@ String generateSlaveXml(String id, String java, String vmargs, String args) thro
         return xml;
     }
 
+    /**
+     * Gets the formatted current time stamp.
+     *
+     * @return the formatted current time stamp.
+     */
+    protected String getTimestamp() {
+        return String.format("[%1$tD %1$tT]", new Date());
```

Comment:
This actually comes from the ssh-slaves plugin but I agree. Maybe some facility could be added at core level to help with this?